02Lesson/02Project/index.html

```diff
@@ -1,10 +1,66 @@
+          </p>
+        </article>
+        <p style="font-family: Oswald, Arial, sans-serif; letter-spacing: .2em;">NEWS AROUND THE WORLD:</p>
+        <article>
+          <a href="http://www.thehockeynews.com/news/article/new-ice-cleaning-roomba-could-spell-the-end-of-the-zamboni-at-hockey-arenas">
+            <h2 style="letter-spacing: .25em; color: #089E00; font-family: EngraversGothicBT-Regular; text-transform: uppercase; font-weight: normal; text-align: center;">Self-Driving Zambonis Cut<br>4K Ice Rink Jobs</h2>
+          </a>
+          <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Jake Samsonite had worked at Texas Ice Arena for almost three decades when he was let go, when he was told his job was given to a machine. He is part of the Zamboni Union of America (ZUA), a group that has been lobbying against self-driving zambonis for the last five years when they were first gaining popularity with ice rinks.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Now more popular than ever, ZUA reports that more than four thousand zamboni drivers are without work, thanks in part to being replaced by artificially intelligent machines. Jake says that besides losing....<a href="http://www.thehockeynews.com/news/article/new-ice-cleaning-roomba-could-spell-the-end-of-the-zamboni-at-hockey-arenas"><span style="font-weight: 700; color: #089E00; text-decoration: none; font-size: .875em; font-family: 100%/1.625 Open Sans     ,HelveticaNeue,Helvetica,Arial,sans-serif;">READ MORE</span></a>
+          </p>
+        </article>
+        <article>
+          <a href="http://www.theonion.com/article/christ-returns-to-nba-1996">
+            <h2 style="letter-spacing: .25em; color: #089E00; font-family: EngraversGothicBT-Regular; text-transform: uppercase; font-weight: normal; text-align: center;">Jesus Christ Returns<br>to The NBA</h2>
+          </a>
+          <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Once in a while, there's that player that you think will never have a replacement. Well, in what is being hailed as the second coming of basketball-christ, Jesus has announced he will be coming out of retirement.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The announcement comes a mere three days after his first shocking announcement that he was essentially dead to the National Basketball Association, having been crucified by the media for his controversial views....<a href="http://www.theonion.com/article/christ-returns-to-nba-1996"><span style="font-weight: 700; color: #089E00; text-decoration: none; font-size: .875em; font-family: 100%/1.625 Open Sans     ,HelveticaNeue,Helvetica,Arial,sans-serif;">READ MORE</span></a>
+          </p>
+        </article>
+      </div>
+    </section>
+    <aside>
+      <a href="http://learn101.org/latin.php" id="sponsor"><div style="background-color: black;">

```

Comment:
I wouldn't put parent and child element tags on the same line. That way you get a consistent indentation and find errors more easily.